parameter/DomainConfiguration.cpp

```diff
@@ -590,7 +604,7 @@ CAreaConfiguration* CDomainConfiguration::getAreaConfiguration(uint32_t uiAreaCo
         }
     }
 
-    assert(0);
+    abort();
```

Comment:
To my opinion those 2 assert -> abort substitution should be enough to avoid klockwork critical.